### ISSUE TITLE
Broken GitHub edit link on policies index

### ISSUE THREAD

**Issue by thibaudcolas** [w3c/wai-policies-prototype#565]
On the [Web Accessibility Laws & Policies](https://www.w3.org/WAI/policies/) page, the "Fork & Edit on GitHub" link is broken in the [Help improve this page](https://www.w3.org/WAI/policies/#helpimprove) section at the bottom. GitHub says "404 not found", "Cannot find a valid ref in `main/_policies/index.md`". Digging a bit through how this is all set up, it seems the target repository and file path are correct, but the branch should be `master` rather than `main`. 

Looking in the [wai-website-theme github-file-path.html](https://github.com/w3c/wai-website-theme/blob/main/_includes/github-file-path.html), it defaults to `master` there, so it seems either `include.p.github.branch` or `site.branch` are set incorrectly somewhere? I’m not clear how the two repos interact to set this value.
